config/default/manager_image_patch.yaml

```diff
@@ -10,4 +10,4 @@ spec:
       # Change the value of image field below to your controller image URL
       - image: gcr.io/k8s-staging-cluster-api-gcp/cluster-api-gcp-controller:latest
         name: manager
-        imagePullPolicy: Always
+        imagePullPolicy: Never
```

Comment:
```suggestion
        imagePullPolicy: Always
```

Reply:
Always?

Reply:
dropped any edits of imagePullPolicy in this PR